### ISSUE TITLE
Explosion from elements tooltip is inverted

### ISSUE THREAD

**Issue by MrNoe93** [GregTechCEu/GregTechCEu-1.19#807]
### GregTech CEu Version

1.20.1-1.1.1

### Recipe Viewer Installed

No Recipe Viewer

### Environment

Singleplayer

### Cross-Mod Interaction

No

### Expected Behavior

When the config for machines exploding when exposed to the elements is turned on, there would be no tooltip saying they dont explode when exposed to the elements

### Actual Behavior

The tooltip about no explosions when exposed to the elements shows up when the explosions are turned on.

### Steps to Reproduce

Change config for explosions when exposed to the elements and then look at tooltip.

### Additional Information

TLDR: the tooltip is backwards.  When machines can explode due to the elements, the tooltip saying they cant shows up.  When they cant, the tooltip is gone.
